front/src/components/mmt-accommodation-card/mmt-accommodation-card.component.js

```diff
@@ -0,0 +1,30 @@
+import {
+  VCard, VCardTitle, VCardSubtitle, VCardText,
+  VCarousel, VCarouselItem, VImg,
+  VRating, VIcon
+} from 'vuetify/lib';
+const { baseUrl } = require('@/config/config.dev.json');
+const { SERVER_ACCOMODATIONS_PATH } = require('@constants');
+
+export default {
+  name: 'mmt-accommodation-card',
+  components: {
+    VCard,
+    VCardTitle,
+    VCardSubtitle,
+    VCardText,
+    VCarousel,
+    VCarouselItem,
+    VImg,
+    VRating,
+    VIcon
+  },
+  props: [ 'accommodation' ],
+  methods: {
+    getFilePath (fileName) {
+      return !fileName.startsWith('http')
+        ? `${baseUrl}/${SERVER_ACCOMODATIONS_PATH}/${fileName}`
+        : fileName;
```

Comment:
i am not sure, but, might, we have already filter for it - if no it good point to make